fiat-roles/src/test/groovy/com/netflix/spinnaker/fiat/permissions/DefaultPermissionsResolverSpec.groovy

```diff
@@ -146,13 +153,41 @@ class DefaultPermissionsResolverSpec extends Specification {
     result == expected
   }
 
+  def "user should be admin"() {
+    fiatAdminConfig.getAdmin().getReadRoles().add("delivery-team")
```

Comment:
It looks like after this test run, the shared fiat admin config is modified, making all remaining tests after this running under a different scenario. Since you're just creating the shared one to avoid NPE, I'd suggest doing something like:

```
.setFiatAdminConfig(new FiatAdminConfig())
``` 
on line 88, and then creating your custom config within this test.